service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -955,6 +961,8 @@ public DynamicList buildDynamicHearingList(List<Element<HearingBooking>> hearing
     private final HearingReListOption hearingReListOption;
     private final HearingCancellationReason adjournmentReason;
     private final HearingCancellationReason vacatedReason;
+    @JsonProperty
```

Comment:
I think you don't need this annotation (unless you'll use a different attribute and field names).